x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -165,7 +173,7 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps & PropsFromRedux
   });
   const { eventFilterActionItems } = useEventFilterAction({
     onAddEventFilterClick: handleOnAddEventFilterClick,
-    disabled: !isEvent,
+    disabled: !isEvent || !canCreateEndpointEventFilters,
```

Comment:
When its disabled due to it not being an event, we should think about adding a tooltip to the item (if supported) indicating that this action is only valid for Events.

Reply:
Also, it feels strange to disable an item if the user does not have Authz to that action. Normally we should hide UI components when that is the case, not disable it.